data/LevelData.json

```diff
@@ -0,0 +1,1058 @@
+{
```

Comment:
We can get rid of this as well when we decide to get rid of the Tools.py file. It'll be handy for now until we solidify how we want to do some of the advanced rando.